### PR TITLE
Changes to security supply chain page.

### PR DESCRIPTION
**Pull request by jhkrug**
There are a few TODO: that I need some input on..

### CODE REVIEW

File: docs/distributing-policies/secure-supply-chain.md [hunk 1]
```diff
@@ -142,22 +163,35 @@ jobs:
...
+
+:::
+
+<!--TODO:
+I need someone to explain what we are trying to say here.
+To be completely clear we should maybe reference the example more directly, say how we found the reference and what it was in the example, why isn't it mandatory and why might you want to do it. Where else and how would you get the image reference?
+-->
+In the example above, we demonstrated using `kwctl` to publish the container image and find the reference to it.
+This is not mandatory.
+You can call `cosign` with the image reference generated elsewhere.
```
Comment (flavio) [inline docs/distributing-policies/secure-supply-chain.md:185]: I think this is a leftover, we can remove it. It doesn't make any sense

File: docs/distributing-policies/secure-supply-chain.md [hunk 2]
```diff
@@ -442,33 +512,30 @@ anyOf: # at least `anyOf.minimumMatches` are required to match
...
-Each of these sections can contain one or more signature requirements.
-Users can also define both sections in one file as shown above. In this situations, all
-the signatures requirements from the `allOf` **AND** a minimum number of matches
-from the `anyOf` section as described in the `minimumMatches` field must be satisfied.
+All the signatures requirements from `allOf` **and** a minimum number from `anyOf` must be met.
 
 ### Signature validation
 
-The users can validate different signatures in the `anyOf` and `allOf` sections.
+<!--TODO:What are we trying to say here?-->
```
Comment (flavio) [inline docs/distributing-policies/secure-supply-chain.md:530]: Certain organization have compliance rules that require a container image/policy to be signed by at least X/Y trusted users or by all the trusted users. This is done to reduce the impact of having one of the trusted users being compromised by an attacker

**Review by flavio — COMMENTED** [review]
Thanks for the changes, this looks better! I've answered your questions

File: docs/distributing-policies/secure-supply-chain.md [hunk 3]
```diff
@@ -1,98 +1,111 @@
...
 
+Signing uses Sigstore's cosign utility.
+At the time of writing, this feature isn't enabled by default in cosign.
+It's still experimental and needs to be explicitly used.
 
-```console
+<!--TODO: Is this still experimental?-->
+```bash
 $ COSIGN_EXPERIMENTAL=1 cosign sign ghcr.io/jvanz/policies/user-group-psp:latest
+```
```
[the remark this answers is not in the document]
Comment (jhkrug) [reply docs/distributing-policies/secure-supply-chain.md:102]: Thanks.

[the remark this answers is not in the document]
Comment (jvanz) [reply docs/distributing-policies/secure-supply-chain.md:102]: I would say to remove the `jvanz` reference there as well. Let's use the `kubewarden` instead.

File: docs/distributing-policies/secure-supply-chain.md (hunk 1 above)
Comment (jhkrug) [reply docs/distributing-policies/secure-supply-chain.md:185]: Thanks.

**Comment by jhkrug** [comment]
> LGTM. But I would say to remove the references to my github user `jvanz` and replace it with the `kubewarden` org.

ok

File: docs/distributing-policies/secure-supply-chain.md (hunk 2 above)
Comment (jhkrug) [reply docs/distributing-policies/secure-supply-chain.md:530]: Thanks. I've had another go at that bit. Needs another review iteration though.

**Review by jvanz — APPROVED** [review]

File: docs/distributing-policies/secure-supply-chain.md [hunk 4]
```diff
@@ -480,29 +530,30 @@ key data in the `key` field. The owner field is optional, but can be useful to c
...
-#### Keyless signatures validation
+### Keyless signature validation
+
+A policy signed in keyless mode doesn't have a public key we can verify.
+You can still verify the policy with the OIDC data used during the signing process.
+For that, it's necessary to define the signature validation as `genericIssuer`.
+
+It's possible to verify information from the signature:
+
+<!--TODO:I would like to check carefully that I have the this correct. Down to the :::note. I have used subject and Subject (Fulcio) to better try and distinguish between subject and Subject in the original text. Does it work? I think we need a concrete example to make it all explicit? I think I may be completely misunderstanding this section.-->
```
Comment (jvanz) [inline docs/distributing-policies/secure-supply-chain.md:541]: It looks good to me.

Comment (jhkrug) [reply docs/distributing-policies/secure-supply-chain.md:541]: Thanks. We'll go with this.

**Pull request merged**